packages/client/src/components/Actions/components/TokenView.js

```diff
@@ -0,0 +1,66 @@
+import { useClipboard, useContacts } from "hooks";
+import Svg from "library/Svg";
+import { getSafeContactName, getTokenMeta } from "utils";
+import ProposedDateView from "./ProposedDateView";
+
+const TokenView = ({ actionView, safeData }) => {
+  const { recipient, vaultId, tokenAmount, timestamp } = actionView;
+  const { name: safeName, address: safeAddress } = safeData;
+
+  const { displayName, tokenType } = getTokenMeta(vaultId) || {};
+
+  const clipboard = useClipboard();
+  const { contacts } = useContacts(safeAddress);
+
+  return (
+    <>
+      <div className="pl-4 mb-5">
+        <span className="columns">Amount</span>
```

Comment:
This span has no children element, what are we trying to achieve with the columns class?

Reply:
Adding this class makes the elements properly aligned, but I can remove it and try an alternative way

Reply:
Doing an element inspection on columns class, it appears that columns does two things: 1. adds gap between the children elements, 2. set the container to `display:flex`. I doubt it has an impact on the alignment. Suggest adding `is-flex is-align-items-center` on the container div If the purpose here is to align the spans.